test/wasm-spec/update.sh

```diff
@@ -17,7 +17,7 @@ fi
 
 for p in $PROPOSALS; do
     printf "Updating proposal ${CYAN}%-22s${NORM} " $p
-    update_proposal_repo $p | $PROGRESS || exit $?
```

Comment:
I'd like to keep these lines as they make the output of running these more concise.